datastore.go

```diff
@@ -21,6 +20,7 @@ func NewDatastore(path string, opt *badger.Options) (*datastore, error) {
 	}
 	opt.Dir = path
 	opt.ValueDir = path
+	// opt.SyncWrites = true // default is false.
```

Comment:
We'll probably set this in IPFS when we construct the datastore (so we can disable it).

Reply:
Done.